src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,35 @@
+<div class="container">
+  <div class="contact">
+    <div class="wrapper logo-wrapper">
+      <img src="../../../img/footer.png" alt="" />
+    </div>
+    <div class="wrapper text-wrapper">
+      <p>
+        〒532-0011　大阪市淀川区西中島6丁目8-23テスト<br />
+        TEL.06-1234-5678 FAX.06-1234-5678　営業時間 / 9:00 - 22:00　定休日 /
+        不定休
+      </p>
+    </div>
+    <div class="icon-list">
+      <i class="fab fa-facebook-f"></i>
+      <i class="fab fa-twitter"></i>
+      <i class="fab fa-instagram"></i>
+      <i class="fas fa-rss"></i>
```

Comment:
それぞれのアイコンも、リンク先に飛ぶ想定なので、aタグで囲みましょう。イメージは以下の通りです。
```suggestion
　  <a href="#">
　      <i class="fab fa-facebook-f"></i>
　  </a>
```